3.10/highlights.md

```diff
@@ -41,6 +41,13 @@ grouped by version
 
 **Enterprise Edition**
 
+- [**EnterpriseGraphs**](graphs-enterprise-graphs.html): A new specialized version of
+  SmartGraphs, with an automatic sharding key selection.
+
+- [**Parallelism for Sharded Graphs**](release-notes-new-features310.html#parallelism-for-sharded-graphs-enterprise-edition):
+  Traversals with many start vertices can now run in parallel, leading to
+  faster results.    
```

Comment:
I miss a description of what graph/deployment types now support it because it was already supported for single server and OneShard clusters (effectively non-sharded graphs) since 3.7.